Makefile

```diff
@@ -17,7 +17,7 @@ HELM_CLIENT_VER_MAJ := $(shell echo ${HELM_CLIENT_VER} | awk -F. '{print $$2}')
 
 # Parameters for deploy tool
 GIT_HEAD := $(shell git rev-list -1 HEAD)
-GIT_LAST_TAG := $(shell git describe --abbrev=0 --tags)
+GIT_LAST_TAG := WRCP_22.12-wrs.7    # version needs to be specified here
```

Comment:
Is there other commit regarding the naming of the deployctl tool? I cannot understand why this tag can affect the name

Reply:
deployctl tools version is obtained in cmd/deploy/cmd/versions.go from the value GIT_LAST_TAG
GIT_LAST_TAG is defined in Makefile and pass it in DEPLOY_LDFLAGS
and refereed in building deployctl
GIT_LAST_TAG is only used for deployctl.
We could hard-cord in cmd/deploy/cmd/versions.go but I prefer to implement this in Makefile because other information for deployctl version are defined in Makefile.